rover-apps/science/src/AdafruitSTEMMAModule.cpp

```diff
@@ -0,0 +1,13 @@
+#include "AdafruitSTEMMAModule.h"
+#include "Module.h"
+#include "mbed.h"
+#include "AdafruitSTEMMA.h"
+
+AdafruitSTEMMAModule::AdafruitSTEMMAModule() {
+    moistureSensorModule = new AdafruitSTEMMA(TEMP_MOIST_I2C_SDA, TEMP_MOIST_I2C_SCL);
+    bool resetStatus = moistureSensor -> reset();
```

Comment:
```suggestion
    bool resetStatus = moistureSensor.reset();
```
moistureSensor is `AdafruitSTEMMA` object so you can directly call its member functions